src/components/MediaRecorder/transcode/audioProcessing.ts

```diff
@@ -0,0 +1,35 @@
+import { readFileAsArrayBuffer } from '../../ReactFileUtilities';
+
+/**
+ * In the context of resampling audio data, AudioContext is used to decode the input audio file into an AudioBuffer,
+ * which is a fundamental data structure representing audio data.
+ * @param file
+ */
+export const toAudioBuffer = async (file: File) => {
+  const audioCtx = new AudioContext();
```

Comment:
Should we close this context eventually? Does this create a leak?

Reply:
Fixed by closing the context: https://github.com/GetStream/stream-chat-react/pull/2339/commits/dab90e282f0c14656d64191e05c1175189b19d8a

Reply:
Check this: https://github.com/GetStream/stream-chat-react/commit/dab90e282f0c14656d64191e05c1175189b19d8a#comments